src/org.learningequality.Kolibri.SearchProvider.service

```diff
@@ -0,0 +1,3 @@
+[D-BUS Service]
+Name=org.learningequality.Kolibri.SearchProvider
+Exec=/usr/bin/flatpak run --command=/app/bin/kolibri_search_service.sh org.learningequality.Kolibri
```

Comment:
This should be a normal `Exec` line, `flatpak` converts it into a `flatpak` command itself, I'm surprised this would ever work as-is.

Reply:
Thanks for the note! In fact, you're right, it didn't yet work! :) I was uploading the in-progress work to the PR so I could point someone to it to get advice. New version that works coming shortly!